cmd/edv-rest/startcmd/start.go

```diff
@@ -188,28 +188,17 @@ func startEDV(parameters *edvParameters) error {
 }
 
 func setLogLevel(userLogLevel string) {
-	var logLevelToUse = 3
-
-	switch strings.ToLower(userLogLevel) {
-	case "":
-		logger.Infof(`No log level set. Defaulting to "info".`)
-	case logLevelCritical:
-		logLevelToUse = 0
-	case logLevelError:
-		logLevelToUse = 1
-	case logLevelWarn:
-		logLevelToUse = 2
-	case logLevelInfo: // already set to 3
-	case logLevelDebug:
-		logger.Infof(`Log level set to "debug". Performance may be adversely impacted.`)
-
-		logLevelToUse = 4
-	default:
+	logLevel, err := log.ParseLevel(userLogLevel)
```

Comment:
I saw that I would be needing to do this string-> log level int conversion in multiple places, so I moved it to a common area. Then I realized the logging library already had a method to do this, so I used that instead. There it requires "warning" instead of "warn" - hence the change to the logLevelWarn constant above as well.